pkgs/servers/monitoring/fusion-inventory/default.nix

```diff
@@ -68,6 +54,8 @@ perlPackages.buildPerlPackage rec {
     XMLTreePP
   ];
 
+  doCheck = false;
```

Comment:
Can you add a comment why the tests are disabled?

Also it would be good if we first merge this into master and then backport.